### PR TITLE
feat(env-checker): fix missing env checker popup when run f5 from WSL

### PR DESCRIPTION
**Pull request by meifans**
fix [bugs](https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/9951141)

### CODE REVIEW

File: packages/fx-core/package.json [hunk 1]
```diff
@@ -60,7 +60,6 @@
     "node-forge": "^0.10.0",
     "node-ts-uuid": "^1.0.8",
     "openapi-types": "^7.2.3",
-    "opn": "^6.0.0",
```
Comment (a1exwang) [inline packages/fx-core/package.json:63]: Also remove "opn" from `packages/vscode-extension/package.json`?

File: packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts [hunk 2]
```diff
@@ -97,6 +97,10 @@ export class VSCodeAdapter implements IDepsAdapter {
     );
     return configuration.get<boolean>(key, false);
   }
+
+  private static async openUrl(url: string): Promise<void> {
+    return commands.executeCommand("vscode.open", Uri.parse(url));
```
Comment (qinezh) [inline packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts:102]: `async` and `await` should be paired?

File: packages/fx-core/package.json (hunk 1 above)
Comment (qinezh) [inline packages/fx-core/package.json:63]: do we need to remove the "opn" from "vscode-extension/package.json" either?

File: packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts (hunk 2 above)
Comment (meifans) [reply packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts:102]: Just a wrapper, I think open url should be a async method.

File: packages/fx-core/package.json (hunk 1 above)
Comment (meifans) [reply packages/fx-core/package.json:63]: Removed

Comment (meifans) [reply packages/fx-core/package.json:63]: Yep, removed.

**Review by a1exwang — APPROVED** [review]

**Comment by a1exwang** [comment]
I always get this error when running `npm setup` after cleanup.

```
lerna notice cli v4.0.0
lerna info versioning independent
lerna info Bootstrapping 7 packages
lerna info Installing external dependencies
lerna ERR! npm install exited 1 in 'teamsfx-extension'
lerna ERR! npm install stderr:
npm ERR! Cannot read property 'match' of undefined

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/alexwang/.npm/_logs/2021-05-18T06_08_23_606Z-debug.log
lerna ERR! npm install exited 1 in 'teamsfx-extension'
lerna WARN complete Waiting for 6 child processes to exit. CTRL-C to exit immediately.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! teamsfx@ bootstrap: `lerna bootstrap`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the teamsfx@ bootstrap script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/alexwang/.npm/_logs/2021-05-18T06_08_23_645Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! teamsfx@ setup: `npm install && npm run bootstrap && npx lerna exec -- npm install --only=dev`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the teamsfx@ setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/alexwang/.npm/_logs/2021-05-18T06_08_23_667Z-debug.log

```

**Review by LongOddCode — APPROVED** [review]

**Review by qinezh — APPROVED** [review]

**Review by kimizhu — APPROVED** [review]

**Review by blackchoey — APPROVED** [review]

**Review by chagong — APPROVED** [review]

**Pull request merged**